client/devdocs/design/index.jsx

```diff
@@ -83,44 +97,44 @@ export default React.createClass( {
+					<ClipboardButtons componentUsageStats={ componentsUsageStats.clipboardButton } />
+					<Count componentUsageStats={ componentsUsageStats.count } />
+					<CountedTextareas componentUsageStats={ componentsUsageStats.countedTextarea } />
+					<DatePicker componentUsageStats={ componentsUsageStats.datePicker } />
+					<DropZones searchKeywords="drag" componentUsageStats={ componentsUsageStats.dropZone } />
+					<ExternalLink componentUsageStats={ componentsUsageStats.externalLink } />
+					<FeatureGate componentUsageStats={ componentsUsageStats.featureGate } />
+					<FilePickers componentUsageStats={ componentsUsageStats.filePicker } />
+					<FoldableCard componentUsageStats={ componentsUsageStats.foldableCard } />
+					<FormFields searchKeywords="input textbox textarea radio" componentsUsageStats={ this.getFormFiledsStats( componentsUsageStats ) } />
+					<Gauge componentUsageStats={ componentsUsageStats.gauge } />
+					<GlobalNotices componentUsageStats={ componentsUsageStats.globalNotices } />
+					<Gridicons componentUsageStats={ componentsUsageStats.gridicon } />
 					<Headers />
-					<InfoPopover />
-					<InputChrono />
-					<Notices />
-					<PaymentLogo />
-					<Popovers />
-					<ProgressBar />
-					<Ranges />
-					<Rating />
-					<SearchDemo />
-					<SectionHeader />
-					<SectionNav />
-					<SegmentedControl />
+					<InfoPopover componentUsageStats={ componentsUsageStats.infoPopover } />
+					<InputChrono componentUsageStats={ componentsUsageStats.inputChrono } />
+					<Notices componentUsageStats={ componentsUsageStats.notices } />
+					<PaymentLogo componentUsageStats={ componentsUsageStats.paymentLogo } />
+					<Popovers componentUsageStats={ componentsUsageStats.popover } />
+					<ProgressBar componentUsageStats={ componentsUsageStats.progressBar } />
+					<Ranges componentUsageStats={ componentsUsageStats.range } />
+					<Rating componentUsageStats={ componentsUsageStats.rating } />
+					<SearchDemo componentUsageStats={ componentsUsageStats.search } />
+					<SectionHeader componentUsageStats={ componentsUsageStats.sectionHeader } />
+					<SectionNav componentUsageStats={ componentsUsageStats.sectionNav } />
+					<SegmentedControl componentUsageStats={ componentsUsageStats.segmentedControl } />
 					<SelectDropdown searchKeywords="menu" />
-					<SocialLogos />
-					<Spinner searchKeywords="loading" />
-					<SpinnerLine searchKeywords="loading" />
-					<TimezoneDropdown />
-					<TokenFields />
+					<SocialLogos componentUsageStats={ componentsUsageStats.socialLogo } />
+					<Spinner searchKeywords="loading" componentUsageStats={ componentsUsageStats.spinner } />
+					<SpinnerLine searchKeywords="loading" componentUsageStats={ componentsUsageStats.spinnerLine } />
+					<TimezoneDropdown componentUsageStats={ componentsUsageStats.timezoneDropdown } />
+					<TokenFields componentUsageStats={ componentsUsageStats.tokenField } />
 					<Version />
```

Comment:
Yep we were discussing about this same thing. Initially I thought of doing this in `client/devdocs/design/search-collection.jsx` but maybe mapping here is even better and neat :) Thank you!